eng/Version.Details.xml

```diff
@@ -1,75 +1,75 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.WindowsDesktop.App.Ref" Version="5.0.0-rc.2.20474.5" CoherentParentDependency="Microsoft.NET.Sdk">
+    <Dependency Name="Microsoft.WindowsDesktop.App.Ref" Version="5.0.0-rc.1.20417.4" CoherentParentDependency="Microsoft.NET.Sdk">
```

Comment:
Just do double check, is the downgrading here intentional?

Reply:
I'm not sure.  This was part of the original automated PR: #8725

I think at this point it's fine to let it downgrade, it will eventually be upgraded to a 6.0 version.

@mmitche thoughts?